10.9-libcxx/stable/main/finkinfo/libs/rmods/cran-processx-r.info

```diff
@@ -75,19 +64,12 @@ InstallScript: <<
   
   mkdir -p %i/lib/R/%type_raw[rversion]/site-library
   pushd %b/.. && $BIN_R --verbose CMD INSTALL --library=%i/lib/R/%type_raw[rversion]/site-library processx
-  if [ "%type_num[rversion]" -lt "34" ]; then
-	  install_name_tool -id %p/lib/R/%type_raw[rversion]/site-library/processx/libs/client.dylib %i/lib/R/%type_raw[rversion]/site-library/processx/libs/client.dylib
-	  install_name_tool -id %p/lib/R/%type_raw[rversion]/site-library/processx/libs/processx.dylib %i/lib/R/%type_raw[rversion]/site-library/processx/libs/processx.dylib
-  else
-	  install_name_tool -id %p/lib/R/%type_raw[rversion]/site-library/processx/libs/client.so %i/lib/R/%type_raw[rversion]/site-library/processx/libs/client.so
-	  install_name_tool -id %p/lib/R/%type_raw[rversion]/site-library/processx/libs/processx.so %i/lib/R/%type_raw[rversion]/site-library/processx/libs/processx.so
-  fi
+-*  install_name_tool -id %p/lib/R/%type_raw[rversion]/site-library/processx/libs/client.so %i/lib/R/%type_raw[rversion]/site-library/processx/libs/client.so
```

Comment:
Stray `-*` characters there at the front?

Reply:
Fixed it.